manifests/cf-manifest/operations.d/229-cf-update_cflinuxfs_releases.yml

```diff
@@ -3,6 +3,6 @@
   path: /releases/name=cflinuxfs3
   value:
     name: "cflinuxfs3"
-    version: "0.94.0"
-    url: "https://bosh.io/d/github.com/cloudfoundry/cflinuxfs3-release?v=0.94.0"
-    sha1: "c005fa270834b50eb1a5fb689cad8e6442f3cfbd"
+    version: "0.96.0"
+    url: "https://bosh.io/d/github.com/cloudfoundry/cflinuxfs3-release?v=0.96.0"
```

Comment:
Is this override needed [upstream seems to have it](https://github.com/cloudfoundry/cf-deployment/blob/v9.3.0/cf-deployment.yml#L2357)?

Reply:
Good shout - I've removed this opsfile